### PR TITLE
demo.training

### PR DESCRIPTION
**Pull request by othan1**
### TODO
- [ ] something
- [ ] something else

### CODE REVIEW

**Comment by othan1** [comment]
This is a comment.  You can tell by the spelling.

File: hello-world.cc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main()
+{
+  std::cout << "Hello, world!\n";
```
Comment (githubteacher) [inline hello-world.cc]: @othan1 Should we think about printing a different message?

File: hello-world.cc [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+}
```
Comment (girish44) [inline hello-world.cc]: @githubteacher ... How about a Wow!!!

File: hello-world.cc (hunk 1 above)
Comment (KalpanaNagarajan) [inline hello-world.cc]: std::cout <<"  First Line of the code";

File: hello-world.cc (hunk 2 above)
Comment (githubteacher) [inline hello-world.cc]: @girish44 Great idea!

File: hello-world.cc (hunk 1 above)
Comment (mnguyen17950) [inline hello-world.cc]: Sure.  Why not.

File: hello-world.cc (hunk 2 above)
Comment (mnguyen17950) [inline hello-world.cc]: Wonderful.

**Comment by othan1** [comment]
Comment is created per requirements.  No change necessary.

File: hello-world.cc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main()
+{
```
Comment (girish44) [inline hello-world.cc]: # Is C is the fastest run time language still !!!! 

#include<stdio.h>

	int main()

	{

		printf(“Hello World\n”);

		return 0;

	}

**Comment by em-dmcgrew** [comment]
Do or do not. There is no try.

**Comment by em-styagi** [comment]
We need this command line to cover from starting

**Pull request merged**